nixos/modules/services/audio/navidrome.nix

```diff
@@ -27,7 +27,16 @@ in
 
       enable = mkEnableOption "Navidrome music server";
 
-      package = mkPackageOption pkgs "navidrome" { };
+      package = mkPackageOption pkgs "navidrome" {
+        example = literalExpression ''
+          pkgs.navidrome.override {
+            plugins = with pkgs.navidromePlugins; [
+              wikimedia
+              coverartarchive
+            ];
+          }
+        ''
```

Comment:
```suggestion
        '';
```

Reply:
thanks was on another branch and just yolo-committing, hoping to give this deeper attention soon.